packages/launchpad/src/setup-actions.ts

```diff
@@ -1,7 +1,10 @@
+import { updateLaunchpadSettings } from '@automattic/data-stores';
 import { isMobile } from '@automattic/viewport';
 import { addQueryArgs } from '@wordpress/url';
 import type { LaunchpadTaskActionsProps, Task } from './types';
 
+const TASKS_TO_COMPLETE_ON_CLICK = [ 'add_about_page' ];
```

Comment:
Nice idea! I want to move "migrate content" task here, now. :-) https://github.com/Automattic/wp-calypso/pull/80361